css/component_styles/list.css

```diff
@@ -1,3 +1,48 @@
 @import url('./components.css');
 
-/* -------- list css -------- */
\ No newline at end of file
+/* -------- list css -------- */
+
+.list-container{
+    background-color: var(--bg-color);
+    width: 100%;
+    padding: 1rem;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+/* normal list */
+
+.list ol, .list ul{
+    padding-left: 40px; 
```

Comment:
This isnt required as the <ul> and <ol> will have a default value of _padding-left :40px_
![image](https://user-images.githubusercontent.com/67600170/160440890-9476e816-dec2-40b4-aa00-92f3d28f027e.png)


Unless you had reset in the reset css. You can remove this.

Reply:
yes i have reset it thats why